benchmarks/imagenet/resnet50/finetune_eval.py

```diff
@@ -5,36 +5,13 @@
 from pytorch_lightning.callbacks import DeviceStatsMonitor, LearningRateMonitor
 from pytorch_lightning.loggers import TensorBoardLogger
 from torch.nn import Module
-from torch.optim import SGD
 from torch.utils.data import DataLoader
 from torchvision import transforms as T
 
 from lightly.data import LightlyDataset
 from lightly.transforms.utils import IMAGENET_NORMALIZE
-from lightly.utils.benchmarking import LinearClassifier, MetricCallback
+from lightly.utils.benchmarking import FinetuneClassifier, MetricCallback
 from lightly.utils.dist import print_rank_zero
-from lightly.utils.scheduler import CosineWarmupScheduler
-
-
-class FinetuneEvalClassifier(LinearClassifier):
-    def configure_optimizers(self):
-        parameters = list(self.classification_head.parameters())
-        parameters += self.model.parameters()
```

Comment:
I believe the only difference between `FinetuneEvalClassifier` and `LinearClassifier` is the learning rate. I think it would be better to expose the learning rate as argument in `LinearClassifier.__init__` than to create a new `FinetuneClassifier` class in `utils/benchmarking`. Wdyt?

Reply:
Sounds good. Actually, I believe we might want to split them in the future anyways. In the future we should extend the linear classifier to multiple heads with different learning rates. This allows to quickly grid search linear evaluation. This is not possible with the fine-tuning classifier.